samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -70,6 +70,9 @@ const locales = {
 	'zh-Hant-HK - Traditional Chinese, custom Hant font': 'zh-Hant-HK',
 	'vi-VN - Vietnamese, with tallglyph characters': 'vi-VN',
 	'ta-IN - Tamil, custom Indian font': 'ta-IN',
+	'bn-IN - Bengali': 'bn-IN',
+	'hi-IN - Hindi': 'hi-IN',
+	'te-IN - Tegulu': 'te-IN',
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/137440211-ae2b2abc-e137-47b7-8a4e-3dd7123b43ff.png)

Please check the result on O20 with te-IN